### PR TITLE
WINDUP-748: Incident counts appear to be off for some issues in the "Migration Issue Report"

### PR DESCRIPTION
**Pull request by jsight**
https://issues.jboss.org/browse/WINDUP-748

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/1058/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/429/
Test PASSed.

**Comment by Maarc** [comment]
I tested the PR Yesterday and it seemed to work well! I have not reviewed the code.

File: reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/problemsummary/GetProblemSummariesMethod.java [hunk 1]
```diff
@@ -70,16 +74,18 @@ public Object exec(List arguments) throws TemplateModelException
...
             }
-            else
-            {
-                summary.setNumberFound(summary.getNumberFound() + 1);
-            }
+
+            int fileCount = summary.getNumberFound();
             for (FileModel file : classification.getFileModels())
+            {
+                fileCount++;
```
Comment (mbriskar) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/problemsummary/GetProblemSummariesMethod.java]: so for the classifications it always had 1?

File: reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreateMigrationIssuesReportRuleProvider.java [hunk 2]
```diff
@@ -47,11 +43,6 @@ public Configuration getConfiguration(GraphContext context)
         @Override
         public void perform(GraphRewrite event, EvaluationContext context)
         {
-            // get all classifications
-            // get all hints
-            // group them by a title of some kind (classification and hint title)
-            // do something interesting with this information :)
```
Comment (mbriskar) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreateMigrationIssuesReportRuleProvider.java]: it is no more interesting? :)

**Comment by mbriskar** [comment]
so 2 mistakes? Classification would have maximally 1 + classification didn't have ruleID set?

**Comment by mbriskar** [comment]
I think this just needs rebase. I can test it locally if needed

**Comment by jsight** [comment]
Right, those were two issues found here.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/1077/
Test PASSed.

**Pull request merged**